teuthology/run_tasks.py

```diff
@@ -239,7 +275,57 @@ def build_email_body(ctx, stack, sleep_time_sec):
     )
     return (subject.strip(), body.strip())
 
+
+def rocket_message(ctx, message, channels):
```

Comment:
I suggest as a less ambiguous name for this function: `rocket_chat_message`

Reply:
Thanks. Renamed to `rocketchat_send_message`.